File: Code_C_bible/C_source code/ch17/C/TestIf.c
#include <stdio.h>
main() {
   int a = 5;
   int b = 3;
   
   if(a<b) {
      printf(“%d is smaller than %d”,a,b);  // ¤ÓÊÑè§ printf à»ç¹¡ÒÃÊÑè§¾ÔÁ¾ì¢éÍ¤ÇÒÁÍÍ¡·Ò§¨ÍÀÒ¾
   }
   else if(a==b) {
      printf(“%d is equal to %d”,a,b);
   }
   else {
      printf(“%d is greater than %d”,a,b);
   }
}

File: Code_C_bible/C_source code/ch17/PHP/TestIf.php
<!DOCTYPE HTML>
<html>
<head>
<title>Test If</title>
</head>
<body>
<?php
   // ประกาศตัวแปร a  และ b
   $a="5";
   $b="3";

   // คำสั่งเงื่อนไข if ซ้อน if
   if ($a<$b) {
       // คำสั่ง echo คือ การส่งข้อความผลลัพธ์กลับไปแสดงผลที่เว็บบราวเซอร์ของเครื่องผู้ใช้งาน
      echo $a."is smaller than".$b; 
   }
    elseif($a>$b) { 
       echo $a."is equal to ".$b;
   }
   else { 
      echo $a."is greater than ".$b;
   }             
?>
</body>
</html>